pkg/services/serviceaccounts/serviceaccounts.go

```diff
@@ -7,13 +7,29 @@ import (
 	"github.com/grafana/grafana/pkg/services/user"
 )
 
-// this should reflect the api
+/*
+ServiceAccountService is the service that manages service accounts.
+
+Service accounts are used to authenticate API requests. They are not users and
+do not have a password.
+*/
 type Service interface {
 	CreateServiceAccount(ctx context.Context, orgID int64, saForm *CreateServiceAccountForm) (*ServiceAccountDTO, error)
 	DeleteServiceAccount(ctx context.Context, orgID, serviceAccountID int64) error
 	RetrieveServiceAccountIdByName(ctx context.Context, orgID int64, name string) (int64, error)
 }
 
+/*
+Store is the database store for service accounts.
+
+migration from apikeys to service accounts:
+HideApiKeyTab is used to hide the api key tab in the UI.
+MigrateApiKeysToServiceAccounts migrates all API keys to service accounts.
+MigrateApiKey migrates a single API key to a service account.
+
+// only used for interal api calls
+RevertApiKey reverts a single service account to an API key.
+*/
 type Store interface {
```

Comment:
The store interface should not be exposed in the domain package. If we want to interact with service accounts the service should be injected. But I think that refactor should be done in another pr

Reply:
il add a new issue for refactoring the store